Build via Docker image fails: `error: failed to parse lock file at: /workspace/thirdparty/milvus/internal/core/thirdparty/tantivy/tantivy-binding/Cargo.lock`

To debug it, I've prepared the container via (from `scripts` directory):

```
docker build -t milvus-lite -f Dockerfile.manylinux.x86_64 .
```

Then entered it via (from top directory):

```
docker run -it -v .:/workspace -v conan:/root/.conan milvus-lite
```

and inside:

```
cd python
python -m build -w
```

The build eventually fails with:

```
[ 39%] Compiling tantivy binding
cd /workspace/thirdparty/milvus/internal/core/thirdparty/tantivy/tantivy-binding && CARGO_TARGET_DIR=/workspace/python/build/bdist.linux-x86_64/build_milvus MACOSX_DEPLOYMENT_TARGET=11.0 cargo build --release
error: failed to parse lock file at: /workspace/thirdparty/milvus/internal/core/thirdparty/tantivy/tantivy-binding/Cargo.lock

Caused by:
  lock file version `4` was found, but this version of Cargo does not understand this lock file, perhaps Cargo needs to be updated?
gmake[2]: *** [CMakeFiles/tantivy_binding_target.dir/build.make:76: compile_tantivy] Error 101
```

--- a/scripts/Dockerfile.manylinux.aarch64
+++ b/scripts/Dockerfile.manylinux.aarch64
@@ -15,7 +15,7 @@ FROM quay.io/pypa/manylinux_2_28_aarch64:2024.07.07-1
 RUN  yum install -y openblas-devel libatomic-static git && cp /usr/lib/gcc/aarch64-redhat-linux/8/libatomic.a /usr/lib64/
 ENV PATH="/opt/_internal/cpython-3.8.19/bin:${PATH}"
 RUN pip3 install conan==1.63.0
-RUN curl https://sh.rustup.rs -sSf | bash -s -- --default-toolchain=1.73 -y
+RUN curl https://sh.rustup.rs -sSf | bash -s -- --default-toolchain=1.89 -y
 ENV PATH="/root/.cargo/bin:${PATH}"
 ENV CCFLAGS="-Wno-error=address"
 ENV CXXFLAGS="-Wno-error=address"

--- a/scripts/Dockerfile.manylinux.x86_64
+++ b/scripts/Dockerfile.manylinux.x86_64
@@ -15,7 +15,7 @@ FROM quay.io/pypa/manylinux_2_28_x86_64:2024.07.07-1
 RUN  yum install -y openblas-devel libatomic-static git && cp /usr/lib/gcc/x86_64-redhat-linux/8/libatomic.a /usr/lib64/
 ENV PATH="/opt/_internal/cpython-3.8.19/bin:${PATH}"
 RUN pip3 install conan==1.63.0
-RUN curl https://sh.rustup.rs -sSf | bash -s -- --default-toolchain=1.73 -y
+RUN curl https://sh.rustup.rs -sSf | bash -s -- --default-toolchain=1.89 -y
 ENV PATH="/root/.cargo/bin:${PATH}"
 
 RUN mkdir -p /workspace

